book.bib

```diff
@@ -549,7 +549,7 @@ @Manual{R-TreeSummarizedExperiment
 }
 
 @Manual{R-MultiAssayExperiment,
-  title =	 {MultiAssayExperiment: Software For The Integration Of Multi-Omics
+  title =	 {{MultiAssayExperiment}: Software For The Integration Of Multi Omics
```

Comment:
The original publication seems to use word "multiomics" https://pubmed.ncbi.nlm.nih.gov/29092936/ 

-> fix